flutter-for-android.md

```diff
@@ -1167,9 +1167,24 @@ The equivalent to Android's density buckets are:
 
 Assets on Flutter can be located in any arbitrary folder; there is no predefined
 folder structure. You then declare where the assets are located in the pubspec
-file, and Flutter will pick them up. Note that Flutter assets are not accessible
-from the native side, and vice versa native assets and resources aren't available
-from Flutter as they live in separate folders.
+file, and Flutter will pick them up.
+
+Note that before Flutter 1.0 beta 2, assets
+defined in Flutter are not accessible from the native side, and vice versa,
+native assets and resources aren't available from Flutter as they live in
+separate folders.
+
+Starting from Flutter 1.0 beta 2, Flutter assets are stored in
```

Comment:
To keep things consistent, let's use 'Flutter beta 2' instead of 'Flutter 1.0 beta 2'

Reply:
d866bc0a3ed0a5efc5fe5288d024773efbaced94